src/ctl/src/common/context.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_rpc_client::MetaClient;
+use risingwave_storage::hummock::HummockStorage;
+use risingwave_storage::monitor::MonitoredStateStore;
+
+use crate::common::hummock_service::{HummockServiceOpts, Metrics};
+use crate::common::meta_service::MetaServiceOpts;
+
+#[derive(Default)]
+pub struct CtlContext {
+    meta_client: tokio::sync::Mutex<CtlContextInner>,
+}
+
+#[derive(Default)]
+struct CtlContextInner {
+    meta_client: Option<MetaClient>,
+    hummock: Option<(MonitoredStateStore<HummockStorage>, Metrics)>,
+}
+
+impl CtlContext {
+    pub async fn get_meta_client(&self) -> anyhow::Result<MetaClient> {
```

Comment:
We can simply name it `meta_client`.

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

```suggestion
    pub async fn meta_client(&self) -> anyhow::Result<MetaClient> {
```

Reply:
Fixed, thanks.